src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -25,6 +24,7 @@
 import frc.robot.utils.ScoringLocationUtil.ScoreHeight;
 import frc.robot.utils.SendableHelper;
 import java.util.TreeMap;
+import frc.robot.subsystems.grabber.*;
```

Comment:
Optimally we don't import asterisks, we import specific files to keep the imports limited to what we actually need. Really minor (especially since these are not exactly huge projects) but it *is* good practice. I'll change it in this case, though.